udata/tests/frontend/test_swagger.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from flask import url_for
+import json
```

Comment:
We put imports from standard lib before the ones from external dependencies following the [PEP8 on imports](https://www.python.org/dev/peps/pep-0008/#imports).